app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
-    pass
+def cache(func: None) -> None:
+    func_results = {}
+
+    def wrapper(*args, **kwargs) -> None:
```

Comment:
`wrapper` won't return None, it will return `Any` value according to the function result.

Reply:
done